packages/paste-website/src/pages/customization/customization-provider.mdx

```diff
@@ -0,0 +1,202 @@
+
+If you&rsquo;re a Flex customer, and want to build a Flex plugin that follows your brand guidelines, the `CustomizationProvider` can be used for that as well.
+
+<ProgrammableAppsCallout />
+
+## About the Customization Provider
+
+Similar to the Paste [ThemeProvider](/tokens/theme-package), the `CustomizationProvider` uses [React Context](https://reactjs.org/docs/context.html) to provide a [theme object](/tokens/theme-package/#theme) to any descendant components in the tree.
+
+The `CustomizationProvider` currently differs in two ways:
+
+1. Firstly, you set a `baseTheme`. The base theme tells the provider which Paste theme you would like to base your custom theme on. You can choose from the 'Default' or 'Dark' Theme.
+2. Secondly, you set a `theme`, but unlike with the `<Theme.Provider />` you pass it an object.
+
+## Selecting a Base Theme
+
+Selecting a base theme allows you to choose which Paste theme you would like to build on top of. By default we select the Default theme for you, but if you would like to create a custom application using a dark theme, you can choose the Dark Base Theme as your starting point.
+
+```
+import {CustomizationProvider} from '@twilio-paste/core/customization';
+
+const MyApp = () => (
+  <CustomizationProvider baseTheme="Dark">
+    rest of app
+  </CustomizationProvider>
+)
+```
+
+## Providing a Theme Object
+
+For the `theme` prop you can pass a complete [theme object](/tokens/theme-package#theme) or a partial theme object for smaller customizations. If a partial theme object is passed, the `CustomizationProvider` merges the custom theme object with the base Paste theme you chose. For example, if you want to customize certain background color tokens of the Dark theme you would only need to pass the following:
+
+```
+import {CustomizationProvider} from '@twilio-paste/core/customization';
+
+const MyApp = () => (
+  <CustomizationProvider baseTheme="Dark" theme={{
+    "backgroundColors": {
+      "colorBackgroundPrimary": "rgb(80, 123, 30)",
+      "colorBackgroundPrimaryDarker": "rgb(56, 86, 21)",
+      "colorBackgroundPrimaryLighter": "rgb(154,199,232)",
+      "colorBackgroundPrimaryLightest": "rgb(197, 223, 242)",
+    }
+  }}>
+    rest of app
+  </CustomizationProvider>
+)
+```
+
+This will output the Paste theme of your choice with the custom background color tokens above. So any component using `colorBackgroundPrimary`, `colorBackgroundPrimaryDarker`, etc will use the custom background color token values instead of the background color token values provided by the base theme.
+
+The theme object should follow the following format, where each theme key can take any valid [design token](/tokens) for that grouping:
+
+<ThemeObjectDisplay />
+
+Custom themes can also be imported as a json object, along with the `CustomizationProvider`. Remember, the custom theme must follow the format of the Paste theme object.
```

Comment:
I don't really understand what this bit is saying. Specifically the `, along with the CustomizationProvider.` part

Reply:
Fair, maybe I can just reword it slightly. It's trying to say you need to import the json object, much in the same way you import the customization provider at the top of your file. But that might not need saying.

Reply:
This might be better